appium-cookies/lib/cookies.js

```diff
@@ -0,0 +1,59 @@
+/*
+ * derived from jQuery Cookie Plugin v1.4.1
+ * https://github.com/carhartl/jquery-cookies
+ */
+ //needed to communicate between JSON protocol cookies and regular js cookies 
+ // doesn't actually store the cookies kind of just translates them 
+import _ from 'underscore';
+import Helper from '../lib/helpers';
+export default class Cookies {
+  constructor (cookieString) {
+  this.ret = cookieString ? cookieString : undefined;
+  this.helper = new Helper();
+  }
+
+  
+  cookie(key, value, options) {
+     if (arguments.length > 1 && !_.isFunction(value)) {
+      options = _.extend({}, options);
+
+      let x = ([
```

Comment:
Outside of things like coordinates, it is probably best to avoid variables named `x`. Is there something more descriptive it can be called?